diagnostic.md

```diff
@@ -55,8 +65,8 @@ const ask = require('../lib/ask.js');
 let answer = 'not empty';
 
 while (answer !== '') {
-  answer = ask("Guess my secret? ");
-  if (answer === 'SeCrEt') {
+  answer = ask("Guess my number? ");
+  for (let i => 1; i <= 10; i++)  {
```

Comment:
You wouldn't do a for loop here. Reason being, you're checking what the user is entering into the terminal so you would need a if statement to check for some criteria, not a for loop.